README.md

```diff
@@ -27,33 +27,12 @@ Step-by-step Deep Leaning Tutorials on Apache Spark using [BigDL](https://github
 + [Setup env on Mac OS](https://github.com/intel-analytics/BigDL-Tutorials/blob/master/SetupMac.md) / [Setup env on Linux](https://github.com/intel-analytics/BigDL-Tutorials/blob/master/SetupLinux.md)
 
 ### Start Jupyter Server
-* You can use pip install BigDL, it will install latest official release version
-    ```
-    pip install BigDL
-    ```
-    then start Jupyter sever
-    ```
-    jupyter notebook --notebook-dir=./ --ip=* --no-browser
-    ```
-* If you want to use different BigDL version, you can download prebuild BigDL binary from https://bigdl-project.github.io/master/#release-download/
-    or you can also download source code and build it yourself, more information please refer https://bigdl-project.github.io/master/#ScalaUserGuide/install-build-src/
-
-   Set BIGDL_HOME variable
-    ```
-    export BIGDL_HOME="Downloaded BigDL located directory"
-    ```
-
-   The please prepare Apache Spark enviroment, dowload Spark from http://spark.apache.org/downloads.html
-
-   Set SPARK_HOME variable
-    ```
-    export SPARK_HOME="Downloaded Spark located directory"
-    ```
-   Use prepared *start_notebook.sh* to start jupyter server
-    ```
-    bash start_notebook.sh
-    ```
+* Run ```pip install BigDL==0.3.0```
```

Comment:
why 0.3.0 version in master branch?

Reply:
I think we forget to update it, has fixed

Reply:
I see. I noticed in both branch-0.3 and branch-0.4, the version is 0.3.0. Please update branch-0.4 as well?